frontend.sh

```diff
@@ -15,9 +15,9 @@ init() {
   source cli-flag.sh 'Front end' $1
 
   if [ -f "package-lock.json" ]; then
-    DEP_CHECKSUM=$(cat package-lock.json package.json | shasum -a 256)
+    DEP_CHECKSUM=$(cat package-lock.json package.json cfgov/unprocessed/apps/**/package*.json | shasum -a 256)
```

Comment:
I supposed based on your edit this could  be `DEP_CHECKSUM=$(cat package*.json cfgov/unprocessed/apps/**/package*.json | shasum -a 256)`

Reply:
good point, fixed in https://github.com/cfpb/cfgov-refresh/pull/3840/commits/c5a8c4cc880c5de2ac66b33972d0515de7fa74ff